src/MiniCover/Commands/Options/WorkingDirectoryOption.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+
+namespace MiniCover.Commands.Options
+{
+    internal class WorkingDirectoryOption : MiniCoverOption<DirectoryInfo>
+    {
+        private const string DefaultValue = "./";
+        private const string OptionTemplate = "--workdir";
+
+        private static readonly string Description = $"Change working directory [default: {DefaultValue}]";
+
+        public WorkingDirectoryOption()
+            : base(Description, OptionTemplate)
+        {
+        }
+
+        protected override DirectoryInfo GetOptionValue()
+        {
+            var workingDirectoryPath = Option.Value() ?? DefaultValue;
+            Directory.CreateDirectory(workingDirectoryPath);
```

Comment:
CreateDirectory return  a directoryInfo and why not returning it at the end

```csharp
        protected override DirectoryInfo GetOptionValue()
        {
            var workingDirectoryPath = Option.Value() ?? DefaultValue;
            Console.WriteLine(workingDirectoryPath);
            var directory = Directory.CreateDirectory(workingDirectoryPath);
            Console.WriteLine($"Changing working directory to '{directory.FullName}'");
            Directory.SetCurrentDirectory(directory.FullName);
            return directory;
        }
```

Reply:
I almost sure that I was changing it for pure code purpose... But still, I don't understand what just happened. Do you have any idea?

Reply:
yes the approval test where not there before and you merged them before commiting your code. So the test avoided to do a regression.